addons/account/tests/common.py

```diff
@@ -10,6 +10,24 @@
 from unittest import SkipTest
 
 
+def instantiate_accountman(cls):
```

Comment:
maybe change for `create_and_set_accountman_as_the_user` ?

And maybe add an optional `companies` parameter to set the companies in the user at the same time?